system_tests/tests/aldn1000.py

```diff
@@ -198,8 +201,11 @@ def test_GIVEN_device_not_connected_WHEN_get_error_THEN_alarm(self):
 
         with self._lewis.backdoor_simulate_disconnected_device():
             self.ca.set_pv_value("STATUS.PROC", 1)
+            #The aladdin scans every 1 second, and seems to react slowly to emulator changes,
+            #tests weren't passing without this sleep
+            sleep(2) 
             self.ca.assert_that_pv_alarm_is('STATUS', ChannelAccess.Alarms.INVALID, timeout=5)
-            
```

Comment:
does increases the value for `timeout=` now work instead of a sleep?

Reply:
Unfortunately not, I tried to increase timeout, but only a sleep() seemed to actually work. I guess sleep works differently enough from timeout?

Reply:
All the assert commands basically call  https://github.com/ISISComputingGroup/EPICS-IOC_Test_Framework/blob/6d2209aa8f8770cc20f910549473d9f5c2ed60b2/utils/channel_access.py#L185 which waits in a loop using `sleep` for up to timeout seconds. The main difference would be that is is sleeping for a small interval and then retrying the read, and keeps retrying until `timeout`  seconds have elapsed and the readback does not agree. So a sleep of 2 or adding 2 to timeout should be the same unless the act of reading after every small sleep is causing an issue. If you changed  https://github.com/ISISComputingGroup/EPICS-IOC_Test_Framework/blob/6d2209aa8f8770cc20f910549473d9f5c2ed60b2/utils/channel_access.py#L209 to 0.1 seconds does that work with a bigger timeout instead?